test/unit/resource.ts

```diff
@@ -99,6 +99,23 @@ describe('Resource', () => {
 
   });
 
+  describe('Resource.ActiveRefreshes', () => {
```

Comment:
This test has very little value and breaks encapsulation. IMO, it should be removed.

Reply:
Done